mmv1/templates/validator/resource_converter_iam.go.erb

```diff
@@ -21,12 +21,13 @@
 
   resource_uri = object.iam_policy.base_url || object.self_link_uri
   parent_resource_name = object.iam_policy.parent_resource_attribute || object.name.underscore
```

Comment:
```suggestion
  parent_resource_name = object.iam_policy.parent_resource_attribute.downcase || object.name.underscore
```

It looks like `underscore` includes a call to `.downcase` so I think setting `asset_resource name` may be duplicative (except for downcasing parent_resource_name in the case where it's not pulled from object.name - I'm not sure whether we need to downcase that? But it probably doesn't hurt either way.)